clang/test/Driver/sycl-offload.c

```diff
@@ -585,6 +585,16 @@
 
 /// ###########################################################################
 
+// RUN: touch %t.a
+// RUN: %clang -fsycl -foffload-static-lib=%t.a -o output_name -lstdc++ -### %s 2>&1 \
```

Comment:
Please add '-z relro' option to the test as well to provide a unique LinkerInput option (that doesn't look like lib)

Reply:
I shall do this, sorry for the bother!